package.json

```diff
@@ -8,12 +8,18 @@
     "url": "https://github.com/apipatth/ppoker-rest-react.git"
   },
   "dependencies": {
+    "@types/jest": "^24.0.23",
+    "@types/node": "^12.12.9",
+    "@types/react": "^16.9.11",
+    "@types/react-dom": "^16.9.4",
+    "@types/react-router-dom": "^5.1.2",
     "classnames": "^2.2.6",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-github-btn": "^1.1.1",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.2.0"
+    "react-scripts": "3.2.0",
+    "typescript": "3.6.4"
```

Comment:
Any reason to avoid using typescript `3.7.X`? 🚀

Reply:
3.7.x is with warnings. Safe to use 3.6.x